Packs/MicrosoftGraphFiles/Integrations/MicrosoftGraphFiles/README.md

```diff
@@ -55,10 +61,6 @@ After you successfully execute a command, a DBot message appears in the War Room
 
 Delete a DriveItem by using its ID
 
-##### Required Permissions
```

Comment:
it would be great if we could for each command mention the required permission, so it will be clear why we actually require the permissions you listed above
you can find it in microsoft api reference